Sources/StreamChatUI/ChatMessageList/ChatMessage/Decorators/ChatUnreadMessagesCountDecorationView.swift

```diff
@@ -25,7 +25,10 @@ open class ChatUnreadMessagesCountDecorationView: ChatMessageDecorationView, The
     override open func updateContent() {
         super.updateContent()
 
-        let unreadCount = content?.unreadCount.messages ?? 0
+        // Temporarily disabling unread counts as they are not 100% accurate all the time.
+        // Passing 0 will show "Unread messages" without a number
+        // let unreadCount = content?.unreadCount.messages ?? 0
+        let unreadCount = 0
```

Comment:
Is there a plan to make this accurate? If yes, shall we have a ticket somewhere about it (and maybe a TODO)?